lib/rubocop/cop/lint/useless_optional_argument.rb

```diff
@@ -0,0 +1,59 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for optional arguments to methods
+      # that do not come at the end of the argument list
+      #
+      # @example
+      #   # bad
+      #   def foo(a = 1, b, c)
+      #   end
+      #
+      #   def bar(a = 1, b, c = 2)
```

Comment:
Good catch. I wrote this before find out that it generates a SyntaxError. I will remove this example.